lib/api/api.dart

```diff
@@ -4,6 +4,8 @@ import 'package:clnapp/model/app_model/list_invoices.dart';
 import 'package:clnapp/model/app_model/list_transaction.dart';
 import 'package:clnapp/model/app_model/pay_invoice.dart';
 
+import '../model/app_model/list_pays.dart';
```

Comment:
```suggestion
import 'package:clnapp/model/model/app_model/list_pays.dart';
```

I do not like the local import, sorry